src/app/routes/index.test.jsx

```diff
@@ -14,8 +21,14 @@ import legacyMediaAssetPage from '#data/azeri/legacyAssets/multimedia/2012/09/12
 import photoGalleryPageJson from '#data/indonesia/cpsAssets/indonesia-41635759.json';
 import storyPageJson from '#data/mundo/cpsAssets/noticias-internacional-51266689.json';
 import featureIndexPageJson from '#data/afrique/cpsAssets/48465371.json';
+import storyPageMostReadData from '#data/pidgin/mostRead/index.json';
 
-afterEach(() => jest.clearAllMocks());
+fetchMock.config.fallbackToNetwork = true; // ensures non mocked requests fallback to an actual network request
```

Comment:
I've never been a fan of relying live or test endpoint in unit tests; it means that if an envinronment is down, your unit tests won't pass, giving a false indication that the code in this unit has broken. Additionally, if you are working offline, your unit tests will fail.

I can't actually see any code that needs access to the network, am I missing something?

Reply:
Previously we mocked the global fetch object which meant that any fetch which occurred always returned the one mock response we had per test (until we reset the mock). Switching to fetchMock allows us to mock multiple fetches based on their url, this showed that our pages make a few different network requests, posts to analytics providers etc and we were only ever mocking the main pagedata.

I would like one to ensure we mock all of our requests however I think that would be quite a large refactor and not in scope for this pr.

Reply:
Oh so this is no 'worse' than the position we are in now on `latest`? Happy to resolve this if that is the case, just wanted to avoid introducing a new dependency on the network if one was being introduced.

Reply:
> Oh so this is no 'worse' than the position we are in now on `latest`? Happy to resolve this if that is the case, just wanted to avoid introducing a new dependency on the network if one was being introduced.

No `worse` at all. This just highlights how many requests we are not yet mocking.